src/Symfony/Component/Ldap/Adapter/ExtLdap/EntryManager.php

```diff
@@ -121,4 +121,26 @@ private function getConnectionResource()
 
         return $this->connection->getResource();
     }
+
+    /**
+     * @param string               $dn
+     * @param iterable|array|ModificationInterface[] $modifications
```

Comment:
I think `iterable|ModificationInterface[]` is sufficient here. the indent is a bit off though